code generators/de.fhdo.lemma.model_processing.code_generation.container_base/src/main/java/de/fhdo/lemma/model_processing/code_generation/container_base/file/docker/DockerComposeService.xtend

```diff
@@ -3,41 +3,35 @@ package de.fhdo.lemma.model_processing.code_generation.container_base.file.docke
 import java.util.Map
 import java.util.List
 import org.eclipse.xtend.lib.annotations.Accessors
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.annotation.JsonProperty
 
 /**
- * This POJO represents the data structure of a service in a docker-compose
- * file in yaml format. It is used by the snakeyaml framework for parsing an existing docker
- * compose file.
+ * This POJO represents the data structure of a docker-compose file in a yaml format used by
```

Comment:
We write proper nouns in their correct capitalization:
- yaml --> YAML
- jackson --> Jackson